assets/src/js/header.js

```diff
@@ -243,8 +246,10 @@ export const setupHeader = () => {
 
   let searchFocused = false;
   const searchInput = document.getElementById('search_input');
+  // eslint-disable-next-line no-unused-expressions
```

Comment:
I think that here instead of disabling ESLint we could refactor using `if` statements, such as:
```
if (searchInput) {
  ...
}
```